store/sqlstore/sqlx_wrapper.go

```diff
@@ -35,17 +35,24 @@ func (w *StoreTestWrapper) DriverName() string {
 	return w.orig.DriverName()
 }
 
+type Builder interface {
+	ToSql() (string, []interface{}, error)
+}
+
 // sqlxExecutor exposes sqlx operations. It is used to enable some internal store methods to
 // accept both transactions (*sqlxTxWrapper) and common db handlers (*sqlxDbWrapper).
 type sqlxExecutor interface {
 	Get(dest interface{}, query string, args ...interface{}) error
+	GetBuilder(dest interface{}, builder Builder) error
```

Comment:
I've only extended a handful of methods -- the ones I needed in the `ThreadStore` -- but it's clearly easy to do if we need them elsewhere.